frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -198,7 +205,7 @@ function StartingJobModal(props: StartingJobModalProps) {
 }
 
 export function NucleiInferralModal({ handleClose }: Props) {
-  const dataset = useSelector((state: OxalisState) => state.dataset);
+  const dataset = useSelector(state => state.dataset);
```

Comment:
These type annotations are not needed. At least according to my local flow checker